src/components/AnonymousChat/child/ChatReplyPreview.tsx

```diff
@@ -11,6 +12,7 @@ import {SIGNED} from '../../../hooks/core/constant';
 import {ScrollContext} from '../../../hooks/screen/useChatScreenHook';
 import {calculateTime} from '../../../utils/time';
 import {fonts, normalizeFontSize} from '../../../utils/fonts';
+import IconVideoPlay from '../../../assets/icon/IconVideoPlay';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `IconVideoPlay` is new in this hunk. Make sure that the SVG icon is correctly exported and can be used without any issues.
```diff
+ import IconVideoPlay from '../../../assets/icon/IconVideoPlay';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->